wpiutil/src/main/native/include/wpi/protobuf/Protobuf.h

```diff
@@ -91,140 +172,14 @@ concept ProtobufSerializable = requires(
 template <typename T>
 concept MutableProtobufSerializable =
     ProtobufSerializable<T> &&
-    requires(T* out, const google::protobuf::Message& msg) {
-      Protobuf<typename std::remove_cvref_t<T>>::UnpackInto(out, msg);
+    requires(T* out, wpi::ProtoInputStream& istream) {
+      Protobuf<typename std::remove_cvref_t<T>>::UnpackInto(out, istream);
     };
 
-/**
- * Unpack a serialized protobuf message.
- *
- * @tparam T object type
- * @param msg protobuf message
- * @return Deserialized object
- */
-template <ProtobufSerializable T>
-inline T UnpackProtobuf(const google::protobuf::Message& msg) {
```

Comment:
What happened to these (public facing) methods?

Reply:
Those were all related to the old serialization method, which is no longer supported after this. They were only here to do a layer of abstraction that is no longer necessary.